packages/manager/modules/pci/src/projects/project/storages/databases/database/general-information/general-information.html

```diff
@@ -344,6 +344,47 @@
                         </oui-action-menu-item>
                     </oui-action-menu>
                 </oui-tile-definition>
+                <oui-tile-definition
+                    data-term="{{:: 'pci_databases_general_information_kafka_rest_api_label' | translate }}"
+                    data-ng-if="$ctrl.isFeatureActivated('restApi')"
+                >
+                    <oui-tile-description>
+                        <p
+                            data-translate="pci_databases_general_information_kafka_rest_api_content"
+                            data-translate-values="{
+                                projectLink: '<a href=&quot' + $ctrl.KARAPACE_URL + '&quot>Karapace</a>'
+                            }"
+                        ></p>
+                        <oui-field>
+                            <oui-switch
+                                class="d-inline-block"
+                                name="rest-api-switch"
+                                model="$ctrl.enableRestApi"
+                                on-change="$ctrl.onRestApiStatusChange(modelValue)"
+                                data-disabled="$ctrl.loading.restApi"
+                            >
+                            </oui-switch>
+                            <label
```

Comment:
**nitpick:** According to our guidelines, labels shouldn't be inline.
see: https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-switch-webcomponent--validation#validation

Reply:
As per showing Enabled/Disabled text next to oui-switch, the same design is followed in other places as well (web.domain). So I have changed it from `label` to `span`.

Reply:
Unfortunately Web Domain section isn't the most recent section.

For consistency you can refer to following sections:
Public Cloud / Databases / (MongoDB|kafkaMirrorMaker) / (Logs|Metrics) tabs